src/client/requester.cpp

```diff
@@ -137,3 +137,170 @@ std::string Requester::getPublicStats(std::string type) {
+    std::vector<std::string> body;
+    std::string path = "private/total-earnings-game/" + session + "/" + host +
+      "/" + gametype;
+    std::string url = baseUrl + path;
+    request.setOpt(new curlpp::options::Url(url));
+    request.setOpt(new curlpp::options::WriteStream(&response));
+    request.perform();
+    while (response >> tmp) {
+      body.push_back(tmp);
+    }
+    return body[body.size() - 1];
+}
+
+std::string Requester::getTotalEarningsPlayer(std::string &session,
+  std::string &host, std::string &player) {
+    std::stringstream response;
+    std::string tmp;
+    std::vector<std::string> body;
+    std::string path = "private/total-earnings-player/" + session + "/" + host
+      + "/" + player;
+    std::string url = baseUrl + path;
+    request.setOpt(new curlpp::options::Url(url));
+    request.setOpt(new curlpp::options::WriteStream(&response));
+    request.perform();
+    while (response >> tmp) {
+      body.push_back(tmp);
+    }
+    return body[body.size() - 1];
+}
+
+std::string Requester::getTotalWinsAll(std::string &session,
+  std::string &host) {
+    std::stringstream response;
+    std::string tmp;
+    std::vector<std::string> body;
+    std::string path = "private/total-wins-all/" + session + "/" + host;
+    std::string url = baseUrl + path;
+    request.setOpt(new curlpp::options::Url(url));
+    request.setOpt(new curlpp::options::WriteStream(&response));
+    request.perform();
+    while (response >> tmp) {
+      body.push_back(tmp);
+    }
+    return body[body.size() - 1];
+}
+
+std::string Requester::getTotalWinsGame(std::string &session,
+  std::string &host, std::string &gametype) {
+    std::stringstream response;
+    std::string tmp;
+    std::vector<std::string> body;
+    std::string path = "private/total-wins-game/" + session + "/" + host +
+      "/" + gametype;
+    std::string url = baseUrl + path;
+    request.setOpt(new curlpp::options::Url(url));
+    request.setOpt(new curlpp::options::WriteStream(&response));
+    request.perform();
+    while (response >> tmp) {
+      body.push_back(tmp);
+    }
+    return body[body.size() - 1];
+}
+
+std::string Requester::getTotalWinsPlayer(std::string &session,
+  std::string &host, std::string &player) {
+    std::stringstream response;
+    std::string tmp;
+    std::vector<std::string> body;
+    std::string path = "private/total-wins-player/" + session + "/" + host +
+      "/" + player;
+    std::string url = baseUrl + path;
+    request.setOpt(new curlpp::options::Url(url));
+    request.setOpt(new curlpp::options::WriteStream(&response));
+    request.perform();
+    while (response >> tmp) {
+      body.push_back(tmp);
+    }
+    return body[body.size() - 1];
+}
+
+std::string Requester::getTotalLossesAll(std::string &session,
+  std::string &host) {
+    std::stringstream response;
+    std::string tmp;
+    std::vector<std::string> body;
+    std::string path = "private/total-losses-all/" + session + "/" + host;
```

Comment:
Some code clones here, we should refactor this section later on, I'll make an issue.